stream-parent/stream-client/src/test/java/org/interledger/stream/sender/SendMoneyAggregatorTest.java

```diff
@@ -52,37 +69,138 @@ public void setUp() {
     when(streamEncryptionServiceMock.encrypt(any(), any())).thenReturn(new byte[32]);
     when(streamEncryptionServiceMock.decrypt(any(), any())).thenReturn(new byte[32]);
     when(linkMock.sendPacket(any())).thenReturn(mock(InterledgerRejectPacket.class));
-
+    final StreamConnection streamConnection = new StreamConnection(StreamConnectionId.of("foo"));
```

Comment:
Intellij shows this as unused. Did you mean to use this or the mock?

Reply:
Fixed.